test/parallel/test-process-beforeExit.js

```diff
@@ -21,6 +21,13 @@ function tryListen() {
     .listen(0)
     .on('listening', common.mustCall(function() {
       this.close();
-      process.on('beforeExit', common.mustCall(() => {}));
+      process.on('beforeExit', tryRepeatedTimer);
     }));
 }
+
+const N = 5;
+let n = 0;
```

Comment:
Can these be closed over or something? If we add more tests later, having top level variables would make them inconvenient I guess.